ftdc/cmd/parser.go

```diff
@@ -27,6 +32,26 @@ type gnuplotWriter struct {
 	metricFiles map[string]*os.File
 
 	tempdir string
+
+	options graphOptions
+}
+
+type graphOptions struct {
```

Comment:
Could you document `minTimeSeconds` and `maxTimeSeconds` with a phrase? I think I understand them, but I had to read through the code.